src/main/java/Main.java

```diff
@@ -30,20 +28,22 @@ public static void main(String[] args) throws Exception {
             int choice = Integer.parseInt(scanner.nextLine());
 
             switch (choice) {
-                case CommandConstants.LOGIN_CODE:
+                case CommandConstants.LOGIN_CODE -> {
                     System.out.println("Your username");
                     String inputUsername = scanner.nextLine();
                     System.out.println("password");
                     String inputPassword = scanner.nextLine();
                     LoginCommand loginCommand = new LoginCommand();
-                    loginCommand.execute(inputUsername, inputPassword);
-                    break;
-
-                case CommandConstants.REGISTER_CODE:
+                    loginCommand.execute(inputUsername, inputPassword, "active");
```

Comment:
Why do you need “active” here?
You never execute login command with another values